src/GeneralStateTestsFiller/stCreate2/CREATE2_EOF1InvalidOpcodesFiller.yml

```diff
@@ -0,0 +1,176 @@
+          mstore(0, calldataload(0))
+
+          // initcode: { mstore(0, 0xef0001010002000000) return(23, 9) } }
+          mstore(128, 0x68ef000101000200000060005260096017f30000000000000000000000000000)
+
+          // Add the opcode to the init code
+          let code := shr(248, mload(0))
+          mstore8(136, code)
+
+          // Try to deploy EOF code containing invalid opcode
+          if gt(create2(0, 128, 19, 0), 0) { sstore(code, 1) }
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: # Send invalid opcodes as data
+      - '0x0c'
+      - '0x0d'
+      - '0x0e'
+      - '0x0f'
+      - '0x1e'
+      - '0x1f'
+      - '0x21'
+      - '0x22'
+      - '0x23'
+      - '0x24'
+      - '0x25'
+      - '0x26'
+      - '0x27'
+      - '0x28'
+      - '0x29'
+      - '0x2a'
+      - '0x2b'
+      - '0x2c'
+      - '0x2d'
+      - '0x2e'
+      - '0x2f'
+      - '0x49'
+      - '0x4a'
+      - '0x4b'
+      - '0x4c'
+      - '0x4d'
+      - '0x4e'
+      - '0x4f'
+      - '0x5c'
+      - '0x5d'
+      - '0x5e'
+      - '0x5f'
+      - '0xa5'
+      - '0xa6'
+      - '0xa7'
+      - '0xa8'
+      - '0xa9'
+      - '0xaa'
+      - '0xab'
+      - '0xac'
+      - '0xad'
+      - '0xae'
+      - '0xaf'
+      - '0xb0'
+      - '0xb1'
+      - '0xb2'
+      - '0xb3'
+      - '0xb4'
+      - '0xb5'
+      - '0xb6'
+      - '0xb7'
+      - '0xb8'
+      - '0xb9'
+      - '0xba'
+      - '0xbb'
+      - '0xbc'
+      - '0xbd'
+      - '0xbe'
+      - '0xbf'
+      - '0xc0'
+      - '0xc1'
+      - '0xc2'
+      - '0xc3'
+      - '0xc4'
+      - '0xc5'
+      - '0xc6'
+      - '0xc7'
+      - '0xc8'
+      - '0xc9'
+      - '0xca'
+      - '0xcb'
+      - '0xcc'
+      - '0xcd'
+      - '0xce'
+      - '0xcf'
+      - '0xd0'
+      - '0xd1'
+      - '0xd2'
+      - '0xd3'
+      - '0xd4'
+      - '0xd5'
+      - '0xd6'
+      - '0xd7'
+      - '0xd8'
+      - '0xd9'
+      - '0xda'
+      - '0xdb'
+      - '0xdc'
+      - '0xdd'
+      - '0xde'
+      - '0xdf'
+      - '0xe0'
+      - '0xe1'
+      - '0xe2'
+      - '0xe3'
+      - '0xe4'
+      - '0xe5'
+      - '0xe6'
+      - '0xe7'
+      - '0xe8'
+      - '0xe9'
+      - '0xea'
+      - '0xeb'
+      - '0xec'
+      - '0xed'
+      - '0xee'
+      - '0xef'
+      - '0xf6'
+      - '0xf7'
+      - '0xf8'
+      - '0xf9'
+      - '0xfb'
+      - '0xfc'
+    gasLimit:
+      - 89000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes: {}
+      network:
+        - 'London+3540+3670'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+         nonce: 1
+       b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+         nonce: 1
+         storage: {}
+      # Not possible to deploy EOF code containing invalid opcode
+       f1ecf98489fa9ed60a664fc4998db699cfa39d40:
```

Comment:
Probably not much sense in including this one expectation here, because address after CREATE2 depends on initcode, so different for each tested opcode.